ruby/game/game_spec.rb

```diff
@@ -5,26 +5,24 @@
 
   it "resets guess count on initialization, prompts user 1 to enter word for user 2 to guess" do
     new_game.word_to_guess
-    expect(new_game.word_to_guess).to eq @word
+    expect(new_game.word_to_guess).to eq ""
```

Comment:
You should NEVER be using gets.chomp inside your class!!!!

This method should be accepting a parameter as an argument and then you can test the return.

gets.chomp should ONLY be used for giving inputs.  Not run in a method.